contrib/coredns/policy/policy_test.go

```diff
@@ -0,0 +1,28 @@
+package policy
+
+import (
+	"context"
+	pb "github.com/infobloxopen/policy-box/pdp-service"
+	"github.com/miekg/dns"
+	"testing"
+)
+
+// This function unit tests handlePermit function
+// Inputs are t *testing.T
+// Output is update in t
+func TestHandlePermit(t *testing.T) {
+	p := new(PolicyMiddleware)
+	tw := new(NewLocalResponseWriter)
+	ctx := context.Background()
+
+	r := new(dns.Msg)
+	r.SetQuestion("example.com.", dns.TypeANY)
+	attrs := []*pb.Attribute{&pb.Attribute{Id: "type", Type: "string", Value: "query"}}
+
+	status, err := p.handlePermit(ctx, tw, r, "SECURITY_POLICY_1790", attrs)
```

Comment:
As written, this won't work. You need to add "contrib/coredns/policy" to the TESTABLE variable here:

https://github.com/infobloxopen/themis/blob/master/Makefile#L2

so that this is run when the CI runs.